src/plugin/example_test.go

```diff
@@ -20,5 +20,5 @@ func TestDoesAnnotate(t *testing.T) {
 
 	err := examplePlugin.Run(ctx, fetcher, agent)
 
-	assert.Nil(t, err, "should not error")
+	assert.NoError(t, err, "should not error")
```

Comment:
This should be `require.NoError` according to another linting rule. Odd this isn't being picked up...

Reply:
It's not _required_ because there's no other assertions/code after this statement, unlike in the other test. (This is why `require.NoError` must be used in the other test.)

For consistency, we could use `require.NoError` everywhere, regardless of whether it's strictly necessary.